enzyme/test/Integration/BlasOpt/first.c

```diff
@@ -0,0 +1,42 @@
+enum CBLAS_ORDER {CblasRowMajor=101, CblasColMajor=102};
```

Comment:
add a test line?

Reply:
How? Results are same, it would need to go into the benchmark section later.
Only added it for now to keep track of source examples.

Reply:
look at the other integration tests. There's a line at the top that wlll cause this to be compiled and run (including doing any verification the results are the same assertions)

Reply:
So if you have a fixed input, and thus know the correct output, you can assert it always gives that, even with the optimization

Reply:
Ah sure, I can do that.
I was more wondering how to check that the optimization will actually be applied.
But I guess for that we have llvm tests.